package.json

```diff
@@ -42,11 +42,13 @@
     "gulp-uglify": "1.1.0",
     "gulp-util": "3.0.7",
     "jest-cli": "0.4.18",
+    "jquery": "1.8.3",
```

Comment:
added to project to test controller ONLY.

jQuery is not bundled into application.